build.gradle

```diff
@@ -58,7 +58,7 @@ compileTestGroovy {
     targetCompatibility = JavaVersion.VERSION_1_7
 }
 
-def libertyAntVersion = "1.9.4"
+def libertyAntVersion = "1.9.5-SNAPSHOT"
```

Comment:
You'll need to modify travis.yml and appveyor.yml to clone and build ci.ant before doing its build.